definitions/types/ca-certificate.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "required": [
+    "cert",
+    "create_time",
+    "expiration_time",
+    "sha1_fingerprint"
+  ],
+  "properties": {
+    "cert": {
+      "type": "string",
+      "title": "Certificate",
+      "description": "Certificate for TLS connection in PEM format"
+    },
+    "create_time": {
+      "type": "string",
+      "format": "date-time",
+      "title": "Create Time",
+      "description": "Creation time of the CA Cert",
+      "examples": [
+        "2022-09-25T20:04:15.521636Z"
+      ]
+    },
```

Comment:
How will we update the artifact in the event of cert renewal?

Reply:
That's a great question. Since the rotation of the cert can happen idependently of the infrastructure deploy we'd have to run the package again so the artifact reflects the changes.

One way we could solve for that is by managing the cert lifecycle ourselves. That way we could couple it with the Terraform that produces the artifact so the rotation happens w/ the artifact update. That would put the responsibility of cert rotation on the bundle operator then, or something for MD to track and automate?

🤷

Reply:
Thats a pretty rough user experience. We should talk this through a little more. It sounds like this could also just break a user's db connection without warning.

Reply:
I think that's true for this service and any keys we manage ourselves. Should we just not support TLS atm and sit on this until we have a solve? Another case might be AWS kms keys, when those expire our tf and artifacts are out of sync.

Reply:
My gut says punt for now. With KMS, AWS does some magic for key rotation clients get a KMS reference and AWS updates it behind the scenes.

Reply:
Additionally, we don't expose dbs to the public internet in AWS which provides a layer of security. Does that hold true in GCP?

Reply:
Yes ^ That's actually the argument I made for removing tls for postgresql, since it forced the client to have a cert. Using SSL is to protect against packet sniffing but since the db is hardcoded to be private, it's only defending against attacks from people who have access to the private network. If someone gains that access it probably means one has larger problems. 

I'll close this, then hard-code our 3 GCP bundles to have TLS off. 👍